src/SAML2/XML/saml/AuthenticatingAuthority.php

```diff
@@ -92,7 +90,7 @@ public function toXML(DOMElement $parent = null): DOMElement
             $parent->appendChild($e);
         }
 
-        $e->textContent = $this->authority;
+        $e->textContent = $this->getAuthority();
```

Comment:
Why this change?

Reply:
My bad... I dunno what I was thinking, but it's undone